### DISCUSSION TITLE
Make sure the plugin's dll is  unblocked in it's properties

### DISCUSSION THREAD

**Discussion by johnrxx99** [Rem0o/FanControl.Releases#2232] [General]
Hi, first post so please forgive my ignorance.

I copied the addin to the folder, rebooted the PC the the above warning came up. What am I doing wqrong?

Cheers.

**Comment by Rem0o** [comment]
Right click property on the DLL, should be a checkbox to unblock.

**Comment by johnrxx99** [comment]
Thanks for the reply but the plugin is still not recognised by Fan Control. This is what I did.

I downloaded the  plugin (https://github.com/Mourdraug/FanControl.AsusWMI/releases/download/1.0.13/FanControl.AsusWMI.dll) to my desktop. I opened the Fan Contol file, also on my desktop, open the pluging file and droped the plugin into it, closed the file. I then reopened the file and and made sure the plugin was ticked and closed the file again. Started Fan Control and no plugin recognised.

I restarted the PC, Fan Control opened and the warning came up that the plugin could not be opened. Open the desktop file again made sure the plugin was tick but nada.

So I'm lost. I'm not used to Github or the coding scene. Is there something else I should be doing in Windows 10 to make the plugin be recognised?

Hope you can help. Cheers,. John

**Comment by johnrxx99** [comment]
I had another look and now understand your reply. I checked the unblock in the dll's properties but it now says "Asus WMI could not initialze or has no sensors" My MB is an Asus RogStrix 550-F with an AMD GPU.

Looking on the bright side with out that dll I can see 4 out of 6 fans and the Nvidia GPU. However, the motherboard does not show on the graph curve (or any other), only the GPU and CPU. There are no fans shown for the CPU.

Is there another plugin that may help and show the MB?

Sorry to be a pain. John

**Reply by Rem0o** [reply]
You shouldn't need any plugins.  Did you check the appropriate boxes in sensor settings? Cpu, gpu and motherboard.

**Reply by Rem0o** [reply]
Also, are all your fans plugged in onto the motherboard on separate headers?

**Comment by johnrxx99** [comment]
Hi, yes motherboard, GPU and CPU are ticked on sensor settings, as well as
embedded EC (asus).

However, on curves I don't get a reading for the motherboard, that's why I
wondered if the Asus plugin would help. On the curves drop down I only get
reading for the CPU (2 for the Ryzen CPU) and 1 for the Nvidia GPU. Nothing
for the motherboard.

All the fans are plugged into headers on the MB.

**Comment by johnrxx99** [comment]
I had another look and now understand your reply. I checked the unblock in the dll's properties but it now says "Asus WMI could not initialze or has no sensors" My MB is an Asus RogStrix 550-F with an AMD GPU.

Looking on the bright side with out that dll I can see 4 out of 6 fans and the Nvidia GPU. However, the motherboard does not show on the graph curve (or any other), only the GPU and CPU. There are no fans shown for the CPU.

Is there another plugin that may help and show the MB?

Sorry to be a pain. John

**Reply by Rem0o** [reply]
You might give the hwinfo a shot if you are familiar with hwinfo already.  Should have all your mobo sensors available.  However ot requires that program to be actively running.

**Reply by johnrxx99** [reply]
Yes, I'm already doing that and it's running. It shows some fans that were not shown before but can't access them as no sensor connection. Doesn't solve the lack of motherboard connection though.

**Comment by johnrxx99** [comment]
Yes, I'm already doing that and it's running. It shows some fans that were not shown before but can't access them as no sensor connection. Doesn't solve the lack of motherboard connection though.

**Reply by Rem0o** [reply]
Not sure what you mean by motherboard "connection"?   Mind sharing a screenshot of what you got in FanControl?

**Comment by johnrxx99** [comment]
Screen shot attached, with hwmaonitor running, ahowing all fans. Those on green are running on manual but cannot be attached to curve. I tried to also show the graph curve not showing motherboard but couldn't.
![Screenshot 2024-01-10 102157](https://github.com/Rem0o/FanControl.Releases/assets/65436900/63847fd0-ae8b-4a9b-9980-56ab0fb743cd)

**Reply by Rem0o** [reply]
Ok first thing, three dot menu on each control card, and turn off manual control. That will get you the curve selector. 
Then, what's available in the temperature source for the graph? Can't you just screenshot it?

**Comment by johnrxx99** [comment]
No manaual and curve OK, doh.

Screen shot of graph curve.
![Screenshot 2024-01-10 103409](https://github.com/Rem0o/FanControl.Releases/assets/65436900/1dc00d33-b42a-4125-ab3b-309dc42c0320)

**Reply by Rem0o** [reply]
Did a quick check:

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/fa1f93a2-eb77-4a51-aafd-af359eae120b)

The code for your board's sensors seems incomplete with no valid temperature readings.  So until this is worked on this is what you got. 

The code itself https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/67ff1135158c669fab2cbcdff2086ffa57fe621c/LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs#L3262

**Comment by johnrxx99** [comment]
I assume only Asus can work on that - good to know the board is lacking temperature readings. Are there MB manufactoers that do have all the readings?

Many thanks for your help and good luck. John
